src/index.html

```diff
@@ -20,8 +21,685 @@
       href="styles/index.scss"
     />
   </head>
-
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo-link"
+      >
+        <img
+          class="header__logo-img"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple.html"
```

Comment:
The href attribute uses a hash symbol (#) which usually indicates an anchor link within the same page. If these are meant to link to different pages, consider using the correct file paths.